cmd/tcl/testworkflow-toolkit/commands/clone.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2024 Testkube.
+//
+// Licensed as a Testkube Pro file under the Testkube Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//	https://github.com/kubeshop/testkube/blob/main/licenses/TCL.txt
+
+package commands
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"github.com/kubeshop/testkube/pkg/ui"
+)
+
+func NewCloneCmd() *cobra.Command {
+	var (
+		paths    []string
+		username string
+		token    string
+		authType string
+		revision string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "clone <uri> <outputPath>",
+		Short: "Clone the Git repository",
+		Args:  cobra.ExactArgs(2),
+
+		Run: func(cmd *cobra.Command, args []string) {
+			uri, err := url.Parse(args[0])
+			ui.ExitOnError("repository uri", err)
+			outputPath, err := filepath.Abs(args[1])
+			ui.ExitOnError("output path", err)
+
+			// Disable interactivity
+			os.Setenv("GIT_TERMINAL_PROMPT", "0")
```

Comment:
would be better to use local cmd.Env than setting it globally.

Reply:
To be honest, I'm not sure if the current solution isn't better though - putting it anywhere else will masquerade it, which may be less obvious during maintenance